doc/api/async_hooks.md

```diff
@@ -2,10 +2,17 @@
 
 <!--introduced_in=v8.1.0-->
 
-> Stability: 1 - Experimental
+> Stability: 1 - Experimental. This module will most likely never reach stable
+> state. Please migrate to other APIs.
 
 <!-- source_link=lib/async_hooks.js -->
 
+There are other APIs available to replace `async_hooks` at least for most use
+cases. Migrate/use them instead of using `async_hooks`. For example:
```

Comment:
This feels like it could be better phrased.

```suggestion
Stable APIs can replace `async_hooks` for most use
cases. We recommend using them instead of `async_hooks`. For example:
```

Reply:
@GeoffreyBooth Any reason not to go with the imperative rather than "we recommend" here?

```suggestion
Stable APIs can replace `async_hooks` for most use
cases. Use them instead of `async_hooks`. For example:
```

As an additional optional nit, maybe get rid of the text introducing the example as it is presumably obvious that the code that follow is an example.

```suggestion
Stable APIs can replace `async_hooks` for most use
cases. Use them instead of `async_hooks`.
```

Reply:
I have intentionally not used the `stable` here because `getActiveResourcesInfo` is expermimental. The stable APIs to replace async hooks are rare...